### PR TITLE
GAN-248: Add missing properties from VB6 version of .data files

### PR DESCRIPTION
**Pull request by Saibamen**
This PR not contains `Sonderwuensche` and `Log`

### CODE REVIEW

**Comment by Saibamen** [comment]
Added `Sonderwuensche` and `Log`

**Review by gdl-reif — COMMENTED** [review]
PR still pending - waiting for feedback from partner developer

File: Gandalan.IDAS.WebApi.Data/DTOs/Produktion/BearbeitungDTO.cs [hunk 1]
```diff
@@ -54,5 +54,101 @@ public class BearbeitungDTO
         public bool NichtFreigegeben { get; set; }
         public bool MassXEditierbar { get; set; }
         public bool GroesseEditierbar { get; set; }
+
```
[the remark this answers is not in the document]
Comment (Saibamen) [reply Gandalan.IDAS.WebApi.Data/DTOs/Produktion/BearbeitungDTO.cs:57]: OK, I will move it to wrapper in IBOS1 microservice - it is needed for our integration tests

File: Gandalan.IDAS.WebApi.Data/DTOs/PositionsDaten/PositionsDatenDTO.cs [hunk 2]
```diff
@@ -40,6 +40,14 @@ public class PositionsDatenDTO
         public string VorgangsNummer { get; set; }
         public string DruckDatum { get; set; }
         public string VersandAdresse { get; set; }
-
+
```
[the remark this answers is not in the document]
Comment (Saibamen) [reply Gandalan.IDAS.WebApi.Data/DTOs/PositionsDaten/PositionsDatenDTO.cs:43]: For `KurzZeichen` we always have `"TES_t"`, so we can skip it

[the remark this answers is not in the document]
Comment (Saibamen) [reply Gandalan.IDAS.WebApi.Data/DTOs/PositionsDaten/PositionsDatenDTO.cs:43]: PR for IBOS2 legacy: https://dev.azure.com/gandalan/IBOS2/_git/IBOS2/pullrequest/157
PR for IBOS2 microservice: https://dev.azure.com/gandalan/IDAS/_git/GanMicroservices/pullrequest/158

File: Gandalan.IDAS.WebApi.Data/DTOs/Produktion/EtikettDTO.cs [hunk 3]
```diff
@@ -11,6 +11,7 @@ public class EtikettDTO
         public Guid EtikettGuid { get; set; }
         public string Kuerzel { get; set; }
         public string Text { get; set; }
+        public Guid Kennzeichen { get; set; }
```
Comment (gdl-reif) [inline Gandalan.IDAS.WebApi.Data/DTOs/Produktion/EtikettDTO.cs:14]: Please remove, will be removed from I1 as well (Ticket Devops#6077)

Comment (Saibamen) [reply Gandalan.IDAS.WebApi.Data/DTOs/Produktion/EtikettDTO.cs:14]:
```suggestion
```

**Pull request merged**